week-2/F-logical-operators/exercise2.js

```diff
@@ -4,16 +4,23 @@
   This program calls some functions that are either missing or incomplete.
   Update the code so that you get the expected result.
 */
-
-function isNegative() {}
+var isNegative = -10;
+function isNegative() {
```

Comment:
This function should take an argument to check if it is negative. Look how it is used below and make sure you get the right answers.